cpp/oneapi/dal/table/homogen.hpp

```diff
@@ -119,18 +121,39 @@ class ONEDAL_EXPORT homogen_table : public table {
     }
 #endif
 
-public:
+    /// Creates a new ``homogen_table`` instance from an array.
+    /// The created table shares data ownership with the given array.
```

Comment:
```suggestion
    /// The created table shares data ownership with the given array.
    ///  Information about data mutability in the array is stored in the table.
```
Not sure we need to mention it here, but definitely we need to make it in the spec.

Reply:
I would not say that, this is implementation details. Moreover, it creates confusion as we say that tables are always immutable.